### ISSUE TITLE
ListView groups displayed in Inspect when visual styles are disabled

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#4205]
* .NET Core Version: 
5.0.100-rc.2.20479.15

* Have you experienced this same bug with .NET Framework?: 
No

**Additional data**
Issue is not reproduced on .NET Core 3.1

**Minimal repro**
1. Create application with ListView.
2. Add Group to ListView.
3. Add two items with group and two items without group.
4. Disable visual styles (remove "Application.EnableVisualStyles();" string from Program.cs)
5. Or import attached application ([Issue-4205.zip](https://github.com/dotnet/winforms/files/5515750/Issue-4205.zip))
6. Run application
7. Open Inspect
8. Try to get data about ListView items

**Actual behavior:**
Groups are displayed in Inspect. Second item from group is not displayed in Inspect.
![4205-issue](https://user-images.githubusercontent.com/23376742/98647044-50c90900-2345-11eb-8b94-553045c59d2f.png)

**Expected behavior:**
Groups should not be displayed in Inspect. All items should be available in Inspect.
![4205-correct](https://user-images.githubusercontent.com/23376742/98647056-54f52680-2345-11eb-8b5a-268e778e7919.png)

**Comment by merriemcgaw** [comment]
We should investigate whether CommonControls 5 ListView control even had groups. That seems like it might have been added in v6. In that case this should be by design as long as we don't crash or do something weird.

**Comment by Olina-Zhang** [comment]
Verified this issue in the latest .Net 6.0 SDK build: 6.0.100-preview.4.21181.20, it is fixed: ListView groups are not displayed in Inspect when visual styles are disabled:
![image](https://user-images.githubusercontent.com/26474449/113267563-11598400-9309-11eb-811c-8b3a168a816c.png)
